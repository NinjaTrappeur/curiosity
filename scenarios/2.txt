# This shows that the first user can verify an email address, but not the
# second user.

reset
as alice
user create alice secret alice@example.com --accept-tos
user create bob secret bob@example.com --accept-tos
user do set-email-addr-as-verified bob  # Will succeed.
user do set-email-addr-as-verified bob  # This is already done.

reset
as bob
user create alice secret alice@example.com --accept-tos
user create bob secret bob@example.com --accept-tos
user do set-email-addr-as-verified alice  # Will fail: missing right.
