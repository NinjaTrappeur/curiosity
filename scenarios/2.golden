4: reset
Resetting to the empty state.
5: as alice
Modifying default user.
6: user signup alice secret alice@example.com --accept-tos
User created: USER-1
Signup confirmation email enqueued: EMAIL-1
7: user signup bob secret bob@example.com --accept-tos
User created: USER-2
Signup confirmation email enqueued: EMAIL-2
8: user do set-email-addr-as-verified bob
User successfully updated.
9: user do set-email-addr-as-verified bob
EmailAddrAlreadyVerified
11: reset
Resetting to the empty state.
12: as bob
Modifying default user.
13: user signup alice secret alice@example.com --accept-tos
User created: USER-1
Signup confirmation email enqueued: EMAIL-1
14: user signup bob secret bob@example.com --accept-tos
User created: USER-2
Signup confirmation email enqueued: EMAIL-2
15: user do set-email-addr-as-verified alice
MissingRight CanVerifyEmailAddr
